db/knex_migrations/README.md

```diff
@@ -38,12 +38,12 @@ exports.up = function(knex) {
         table.increments('id');
         table.decimal('price').notNullable();
         table.string('name', 1000).notNullable();
-    }).then(() => {
+    }).then(() => 
```

Comment:
Yes, that's correct. But it's wrong. The `then` statement is not returning it's returned Promise therefore the step is not awaited by Knux.  In this example the inserte statement will not be executed. 

O/c I can revert this change but it means it will remain wrong in the README. Let me know how to proceed here.

Reply:
In that case we can keep it. It looked like an unrelated formatting change on first glance